style.css

```diff
@@ -1,29 +1,90 @@
 body {
-  background-color: #e2e3ed;
+  background-color: #f9f9fb;
+  font-family: "Open Sans", sans-serif;
 }
-.box_form {
-  background-color: white;
-  width: 500px;
-  padding: 50px;
-  box-shadow: 0px 13px 26px rgba(98, 69, 231, 0.15);
-  border-radius: 20px;
+#box {
   display: block;
+  width: 840px;
+  min-height: 100vh;
   margin: 50px auto;
+  border-radius: 20px;
+  background-color: white;
+  box-shadow: 0px 4px 16px rgba(204, 204, 206, 0.5);
 }
+#container_one {
+  display: block;
+  min-height: 100vh;
+  width: 280px;
+  float: left;
+  border-radius: 20px 0px 0px 20px;
+  background-image: url(img/men_calendar.png);
+}
+
+#container_two {
+  display: block;
+  float: left;
+  width: 464px;
+  padding: 48px;
+}
+
+h1 {
+  font-family: "Lora", serif;
+  color: #494a7d;
+  margin: 0px 0px 24px 0px;
+}
+.divider {
+  border: 1px solid #f9f9fb;
+  margin-bottom: 24px;
+}
+
+.container_input input {
+  height: 32px;
+  margin-top: 8px;
+  margin-bottom: 24px;
+  width: 320px;
+  padding-left: 8px;
+}
+
+label {
+  color: #494a7d;
+}
+.form_buttons_container {
+  display: block;
+  float: left;
+  width: 100%;
+}
+.form_buttons_container input {
+  border-width: 0px;
+  border-radius: 4px;
+  padding: 8px;
+  width: 106px;
+  height: 42px;
+  background-color: #0c75ff;
+  color: white;
+  font-weight: 600;
+  font-size: 16px;
+  margin: 8px 8px 32px 0;
+}
+
+/* .cancel_buttons input {
```

Comment:
Con `git` è molto facile tenere traccia del vecchio codice, quindi ha poco senso commentare "in caso ci serva". Eliminare